mep3_behavior_tree/include/mep3_behavior_tree/wait_match_start_topic.hpp

```diff
@@ -0,0 +1,88 @@
+// Copyright 2021 Memristor Robotics
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef MEP3_BEHAVIOR_TREE__WAIT_MATCH_START_TOPIC_HPP_
+#define MEP3_BEHAVIOR_TREE__WAIT_MATCH_START_TOPIC_HPP_
+
+#include <string>
+
+#include "rclcpp/rclcpp.hpp"
+#include "std_msgs/msg/int8.hpp"
+#include "behaviortree_cpp_v3/condition_node.h"
+
+namespace mep3_behavior_tree
+{
+
+    class WaitMatchStart : public BT::ConditionNode
```

Comment:
```suggestion
    class WaitMatchStart : public BT::ActionNodeBase
```
Bas dosta puta stampa fejlove na pocetku. Da bismo to izbjegli jedan nacin je da konvertujemo ovaj BT cvor u BT action. I dalje slusamo `/match_start_status`, ali tick() treba da vrati SUCCESS tek kada je sulov ispunjen.